integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonMetaStore.scala

```diff
@@ -144,6 +145,15 @@ trait CarbonMetaStore {
   def getThriftTableInfo(tablePath: CarbonTablePath)(sparkSession: SparkSession): TableInfo
 
   def getTableFromMetadataCache(database: String, tableName: String): Option[CarbonTable]
+
+  def getCarbonDataSourceHadoopRelation(sparkSession: SparkSession,
+      tableIdentifier: TableIdentifier): CarbonDatasourceHadoopRelation
+
+  def getSchemaFromUnresolvedRelation(sparkSession: SparkSession,
```

Comment:
add comment for interface function

Reply:
This function is simple logic only, why not do this logic directly in caller?

Reply:
This method is called from the parser and this logic for retrieving schema from unresolved relation, according to me should not return in parser. Please let me know if you have a different perspective